### ISSUE TITLE
IDEA

### ISSUE THREAD

**Issue by INFINITY-2000** [dragonfire535/xiao#64]
Hello is it possible a werewolf game in discord ?

**Comment by dragonfire535** [comment]
Yes it's possible, and actually has been done in Xiao before. For now it's staying out until I can find a way to make it without having to await DM messages, which is impossible with sharding.
